vendor/engines/secure_rooms/app/services/secure_rooms/check_access.rb

```diff
@@ -0,0 +1,24 @@
+module SecureRooms
+
+  class CheckAccess
+
+    DEFAULT_RULES = [
+      AccessRules::OperatorRule,
+      AccessRules::AccountSelectionRule,
+      AccessRules::DefaultRestrictionRule,
```

Comment:
I think we should call the last item `DenyRule` or `DenyAllRule`. To me, since we're not handling the last case inside of `authorize`, explicitly saying what the last rule does will probably make it clearer than `default`.